src/Toolkit/Toolkit.WPF/UI/Controls/PopupViewer/AttachmentsPopupElementView.cs

```diff
@@ -0,0 +1,122 @@
+﻿// /*******************************************************************************
+//  * Copyright 2012-2018 Esri
+//  *
+//  *  Licensed under the Apache License, Version 2.0 (the "License");
+//  *  you may not use this file except in compliance with the License.
+//  *  You may obtain a copy of the License at
+//  *
+//  *  http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  *   Unless required by applicable law or agreed to in writing, software
+//  *   distributed under the License is distributed on an "AS IS" BASIS,
+//  *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  *   See the License for the specific language governing permissions and
+//  *   limitations under the License.
+//  ******************************************************************************/
+
+using Esri.ArcGISRuntime.Mapping.Popups;
+using Microsoft.Win32;
+using System.Windows.Controls.Primitives;
+
+namespace Esri.ArcGISRuntime.Toolkit.Primitives
+{
+    /// <summary>
+    /// Supporting control for the <see cref="Esri.ArcGISRuntime.Toolkit.UI.Controls.PopupViewer"/> control,
+    /// used for rendering a <see cref="AttachmentsPopupElement"/>.
+    /// </summary>
+    [TemplatePart(Name ="AttachmentList", Type = typeof(ListBox))]
+    public class AttachmentsPopupElementView : Control
+    {
+        private ListBox? itemsList;
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AttachmentsPopupElementView"/> class.
+        /// </summary>
+        public AttachmentsPopupElementView()
+        {
+            DefaultStyleKey = typeof(AttachmentsPopupElementView);
+        }
+
+        /// <inheritdoc />
+        public override void OnApplyTemplate()
+        {
+            base.OnApplyTemplate();
+            itemsList = GetTemplateChild("AttachmentList") as ListBox;
+            if (itemsList != null)
+            {
+                itemsList.SelectionMode = SelectionMode.Single;
+                itemsList.SelectionChanged += ItemsList_SelectionChanged;
+                LoadAttachments();
+            }
+        }
+
+        private void ItemsList_SelectionChanged(object sender, SelectionChangedEventArgs e)
+        {
+            if(e.AddedItems != null && e.AddedItems.Count > 0)
+            {
+                var attachment = e.AddedItems[0] as PopupAttachment;
+                if(attachment?.Attachment != null)
+                {
+                    OnAttachmentClicked(attachment);
+                }
+                if (sender is Selector s)
+                    s.SelectedValue = null;
```

Comment:
What does this do?

Reply:
It immediately unselects the listview item, so you get a "click" behavior instead of a "select" behavior.